test/common/stats/utility_fuzz_test.cc

```diff
@@ -53,7 +61,8 @@ DEFINE_FUZZER(const uint8_t* buf, size_t len) {
     Stats::Utility::counterFromElements(*scope, {});
   } else {
     // add random length string in each loop
-    while (provider.remaining_bytes() > 3) {
+    size_t iteration = 0;
```

Comment:
nit: suggest for loop with remaining bytes condition over iteration?

Reply:
Done